#goal
long_term_deployment/Task task
----
#result
string success_msg
----
#feedback
string status
